auth/forward.go

```diff
@@ -0,0 +1,58 @@
+package auth
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/types"
+	"github.com/stretchr/stew/objects"
+)
+
+// Forward the authentication to a external server
+func Forward(forward *types.Forward, w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+
+	client := http.Client{}
+
+	forwardReq, _ := http.NewRequest("GET", forward.Address, nil)
+	forwardReq.Header.Add("Accept", "application/json")
+	forwardQuery := forwardReq.URL.Query()
+	rQuery := r.URL.Query()
+	for _, reqParam := range forward.RequestParameters {
+		paramValue := rQuery.Get(reqParam.Name)
+		forwardQuery.Add(reqParam.As, paramValue)
+	}
+	forwardReq.URL.RawQuery = forwardQuery.Encode()
+	forwardResponse, forwardErr := client.Do(forwardReq)
+	if forwardErr != nil || forwardResponse.StatusCode != 200 {
+		log.Debugf("Auth failed...")
```

Comment:
I think forwardErr != nil is not the same as forwardResponse.StatusCode != 200.
Also, you should sent back an error to the client, right?